vic/drivers/shared_image/src/get_global_domain.c

```diff
@@ -31,7 +31,8 @@
  *****************************************************************************/
 size_t
 get_global_domain(char          *nc_name,
-                  domain_struct *global_domain)
+                  domain_struct *global_domain,
```

Comment:
Yes, we can do this when it is not obvious what the parameters are but in this case, I'd argue that you can figure out what `coords_only` is based on the function name and the type.